Fix incorrect doc about the dtype for `torch.randint` described in issue #56347

NivekT opened the pull request:
Summary: Fix incorrect documentation about the dtype for `torch.randint` described in issue #56347

Test Plan: Review documentation to make sure formatting is right

Differential Revision: D29321181

bdhirsh commented on torch/_tensor_docs.py:
```diff
@@ -1840,10 +1840,9 @@ def add_docstr_all(method, docstr):
 
 add_docstr_all('index_put',
                r"""
-index_put(tensor1, indices, values, accumulate=False) -> Tensor
```
Is this PR also trying to fix a different doc issue?

Maybe you just need to rebase on master when your other doc fix lands: https://github.com/pytorch/pytorch/pull/60486/files

bdhirsh reviewed (approved):
You probably just need to rebase before landing

NivekT committed:
Fix incorrect doc about the dtype for `torch.randint` described in issue #56347 (#60507)

Summary:
Pull Request resolved: https://github.com/pytorch/pytorch/pull/60507

Fix incorrect documentation about the dtype for `torch.randint` described in issue #56347

Test Plan: Review documentation to make sure formatting is right

Reviewed By: bdhirsh

Differential Revision: D29321181

fbshipit-source-id: b411308ef707b15845361ab69e9394bc5f110dc0
